modules/kallistobustools/count/main.nf

```diff
@@ -2,18 +2,20 @@ process KALLISTOBUSTOOLS_COUNT {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? 'bioconda::kb-python=0.26.3' : null)
+    conda (params.enable_conda ? 'bioconda::kb-python=0.27.2' : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/kb-python:0.26.3--pyhdfd78af_0' :
-        'quay.io/biocontainers/kb-python:0.26.3--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/kb-python:0.27.2--pyhdfd78af_0' :
+        'quay.io/biocontainers/kb-python:0.27.2--pyhdfd78af_0' }"
 
     input:
     tuple val(meta), path(reads)
     path  index
     path  t2g
     path  t1c
     path  t2c
-    val   workflow_mode
+    val   use_t1c
+    val   use_t2c
```

Comment:
Can't we specify `-c1 $tc1` and `-c2 $tc2` only if the paths are there? instead of having these values?

Reply:
Fair point & good idea. That should work. What I'm wondering mostly is why the `nf-core modules create-test-yml` only created YML for the `kallistbustools/ref` part...

Reply:
@apeltzer @fmalmeida is this the last point missing here?

Reply:
Almost. I am addressing this. There is this point and that the workflow is not correctly connected thus a YAML for the count module is not being created.